packages/schema/src/config/app.ts

```diff
@@ -184,6 +184,21 @@ export default defineUntypedSchema({
      */
     rootTag: {
       $resolve: val => val || 'div'
+    },
+
+    /**
+     * Customize Nuxt root element tag.
+     */
+    teleportTag: {
+      $resolve: val => val || 'div'
+    },
+
+    /**
+     * Customize Nuxt Teleport element id.
+     * @type {string | false}
+     */
+    teleportId: {
+      $resolve: val => val === false ? false : (val || '__nuxt-teleport')
```

Comment:
I don't quite like the DX of:

```vue
<Teleport to="#__nuxt-teleport">
```

Do you think we could improve this? Maybe even just `#teleports` would be better as a default.

Reply:
@danielroe Happy to move the name to `teleports` by default 👍

Reply:
```suggestion
      $resolve: val => val === false ? false : (val || 'teleports')
```